diff --git a/makefile.cargo b/makefile.cargo
index 622583f..ed66644 100644
--- a/makefile.cargo
+++ b/makefile.cargo
@@ -17,7 +17,7 @@ endif
 #                                                                             #
 ###############################################################################
 CXX_FLAGS = "-Wall -Werror -Wunused-but-set-variable"
-CXX_FLAGS += " -fsanitize-coverage=trace-pc-guard"
+
 
 # Debug build flavor of JavaScriptCore
 CMAKE_ARGS := "-DPORT=JSCOnly"
